crates/containerd-shim-wasm/src/sandbox/cli.rs

```diff
@@ -212,7 +212,10 @@ fn shim_main_inner<'a, I>(
             OtlpConfig::set_trace_context(&ctx).unwrap();
         } else {
             let ctx = OtlpConfig::get_trace_context().unwrap();
-            std::env::set_var("TRACECONTEXT", ctx);
+            // SAFETY: we are in a single-threaded context
```

Comment:
This is not really true. When this runs we have already started a miltithreaded tokio runtime.
But I think we can *assume* that all the other threads aren't messing around with env vars at this point.